.github/workflows/pr_automations.yml

```diff
@@ -0,0 +1,78 @@
+# Runs all automations related to PR events.
+#
+# See `issue_automations_init.yml` for the corresponding implementation for
+# issues.
```

Comment:
Does this file exist? If not, we can remove this comment

```suggestion
```

Oh actually, seeing that the same comment is in the other pr automations workflow file. Is this meant to be this instead?

```suggestion
#
# See `issue_automations.yml` for the corresponding implementation for issues.
```

Reply:
Fixed in 53142df. The file exists, the `_init.yml` suffix is incorrect.